homeassistant/components/bluetooth/util.py

```diff
@@ -15,6 +16,38 @@
     WINDOWS_DEFAULT_BLUETOOTH_ADAPTER,
     AdapterDetails,
 )
+from .models import BluetoothServiceInfoBleak
+
+
+async def async_load_history_from_system() -> dict[str, BluetoothServiceInfoBleak]:
+    """Load the device and advertisement_data history if available on the current system."""
+    if platform.system() != "Linux":
+        return {}
+    from bluetooth_adapters import (  # pylint: disable=import-outside-toplevel
+        BlueZDBusObjects,
+    )
+
+    bluez_dbus = BlueZDBusObjects()
+    await bluez_dbus.load()
+    now = time.monotonic()
```

Comment:
Any value in in-lining this, since it's only used once? (purely stylistic and up to you)

Reply:
Since it's a dict comprehension I didn't want to call it every time